lib/linguist/languages.yml

```diff
@@ -4106,7 +4106,7 @@ REALbasic:
   - ".rbres"
   - ".rbtbar"
   - ".rbuistate"
-  tm_scope: source.vbnet
+  tm_scope: source.xojo
```

Comment:
If you're changing the grammar for a language, could you please fill in the corresponding section in the pull request template? In particular, could you add links to Lightshow to show the improvements this change brings?

Reply:
- [x] **I am changing the source of a syntax highlighting grammar**
  - Old: [vbnet](https://github-lightshow.herokuapp.com/?utf8=✓&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fangryant0007%2FVBDotNetSyntax%2Fmaster%2Fvbdotnet.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Fmacoslib%2Fmacoslib%2Fmaster%2Fmacoslib%2FCocoa%2FNSArray.rbbas&code=)
  - New: [xojo](https://github-lightshow.herokuapp.com/?utf8=✓&scope=from-url&grammar_format=auto&grammar_url=https%3A%2F%2Fraw.githubusercontent.com%2Fjimmckay%2FXojoSyntaxTM%2Fmaster%2FXojo.tmLanguage&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fraw.githubusercontent.com%2Fmacoslib%2Fmacoslib%2Fmaster%2Fmacoslib%2FCocoa%2FNSArray.rbbas&code=)

Reply:
Thanks! Please add it to the original post as well, for other reviewers.